tests/NotificationTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+use App\Conference;
+use App\Events\ConferenceCreated;
+use App\Listeners\SendNotificationForOpenCFPs;
+use App\Notifications\CFPIsOpen;
+use App\User;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\Notification;
+
+class NotificationTest extends IntegrationTestCase
+{
+    /** @test */
+    function if_approved_conference_is_created_users_are_notified()
+    {
+        // TEST
+        factory(App\User::class)->states('withNotifications')->create();
+        // TEST
+
+
+        Notification::fake();
+        $user = factory(App\User::class)->states('withNotifications')->create();
+        $conference = factory(Conference::class)->create(['approved' => true]);
+
+        $listener = new SendNotificationForOpenCFPs();
```

Comment:
Can we instead trigger the event and let the eventing system handle passing it up to the listener?